### PR TITLE
Notices

### PR DESCRIPTION
**Pull request by jbamlee**

### CODE REVIEW

File: api/service/notice/dto.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	if p.Limit == 0 {
+		p.Limit = DEFAULT_LIMIT
+	}
+
+	if p.Limit > MAX_LIMIT {
+		p.Limit = MAX_LIMIT
+	}
+
+	if p.Asc && p.After == DefaultPaginationCond.After {
+		p.After = 0
```
Comment (jhlee-young) [inline api/service/notice/dto.go:44]: can this make id `0` one not be displayed?

Comment (jbamlee) [reply api/service/notice/dto.go:44]: > can this make id 0 one not be displayed?

I am sorry could you explain it more?
Do you mean in the query?

Comment (jbamlee) [reply api/service/notice/dto.go:44]: we are using a default setting of Postgresql primary key(integer id).
it starts from the `1`. there will be no problem for id `0` 

I am not sure this answer is correct for your question. let me know if it is wrong.

Comment (jhlee-young) [reply api/service/notice/dto.go:44]: oh I missed that, sorry for confusing

**Review by jhlee-young — APPROVED** [review]

**Pull request merged**